components/Pagination/Pagination.js

```diff
@@ -5,8 +5,6 @@ var defaultsDeep = require('lodash/object/defaultsDeep');
 var Paginator = require('./Paginator');
 var PaginationLink = require('./PaginationLink');
 
-require('style?prepend!raw!./pagination.css');
```

Comment:
All this will disappear. No more CSS in the js build, this will be included in a separate CSS file.

Just create a `themes/default.css` for now and move the content of the file into it (and include it in the example page). We'll have to do that for every widget, but if this is a blocker for you, go ahead :)